docs/recipes/bitbucket-pipelines.md

```diff
@@ -0,0 +1,52 @@
+This document shows how you can setup semantic-release with bitbucket's pipelines and a private npm registry.
+
+# Package.json
+Make sure you configure your package.json file with the required plugins.
+A sample configuration may look like this.
+
+```
+"devDependencies": {
+    "@semantic-release/changelog": "^3.0.2",
+    "@semantic-release/commit-analyzer": "^6.1.0",
+    "@semantic-release/git": "^7.0.7",
+    "@semantic-release/npm": "^5.1.3",
+    "@semantic-release/release-notes-generator": "^7.1.4",
+    "semantic-release": "^15.13.2"
+  },
+  "release": {
+    "repositoryUrl": "<SSH Repo URL>",
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
+      "@semantic-release/changelog",
+      "@semantic-release/git"
+    ]
+  }
+  ```
+  
+# Pipelines Configuration
+1. (Optional) If you're using a private registry follow the steps here https://npme.npmjs.com/docs/tutorials/pipelines.html then set the environment variables NPM_REGISTRY_URL & NPM_TOKEN in the repository's pipelines' settings from bitbucket's UI..
```

Comment:
I removed the first part.

I still think the NPM_REGISTRY_URL & NPM_TOKEN variables are needed for the npm plugin to be able to publish to the registry, since pipelines runs in a clear docker container, it doesnt have all the user related info or an .npmrc file